.github/workflows/test_python.yml

```diff
@@ -59,5 +59,6 @@ jobs:
     - run: |
         sudo apt update
         sudo apt install gcc-${{ matrix.gcc }} g++-${{ matrix.gcc }}
+    - run: pip install tensorflow
```

Comment:
I feel this line triggered the failure in unittests.
@njzjz do you have any hint?

Reply:
let me check...

Reply:
Yes. And for original script, I forgot to add environment variables...

Reply:
seems like I was incorrectly trying to solve the lack of environment variables by installation of TF from pip